index.html

```diff
@@ -40,6 +40,7 @@
             <li><a title='Intro slides' href="http://iamit.in/sympy-pycon/pycon2015.pdf" target="_blank" rel=external>Slides</a></li>
             <li><a title='IPython Notebooks' href="http://nbviewer.ipython.org/github/aktech/sympy-pycon/tree/master/PyCon%20material/tutorial_exercises/" target="_blank" rel=external>IPython Notebooks</a></li>
             <li><a title='HTML Tutorial' href="#" target="_blank" rel=external>HTML Tutorial</a></li>
+            <li><a title='Github Repo' href="http://github.com/aktech/pycon-sympy" target="_blank" rel=external>Source</a></li>
```

Comment:
Though, the link works as of now, but the correct link is: https://github.com/aktech/sympy-pycon